components/forms/cct/CctCalcCreate.tsx

```diff
@@ -192,6 +192,11 @@ export function CctCalcCreate() {
                                     selectedProgramme.endDate,
                                     false
                                   );
+                                  setFieldValue(
+                                    "programmeMembership.designatedBodyCode",
+                                    selectedProgramme.designatedBodyCode,
+                                    false
+                                  );
```

Comment:
@ReubenRobertsHEE 
I was trying to keep things together, to populate DBC data while we are populating other PM data like programme name, start day and end day, so it is saved in the DB when the POST/PUT api call behind (but not show in the UI). Otherwise, we can add it in the BE to find the matched DBC according to the program when the CCT is create/updated.
Please advise if you have any concern about it.

Reply:
I was  just wondering about designatedBody, as opposed to designatedBodyCode. I might be missing something, but it only seems to be mentioned once 😁 .